capstone.css

```diff
@@ -0,0 +1,984 @@
+  display: flex;
+  flex-direction: column;
+  row-gap: 20px;
+  width: 100%;
+}
+
+.program-content {
+  display: flex;
+  width: 100%;
+  position: relative;
+  border: 1px solid grey;
+  align-items: center;
+  justify-content: flex-start;
+  background-color: rgba(7, 7, 7, 0.89);
+  opacity: 0.8;
+  padding: 20px 0;
+  column-gap: 25px;
+}
+
+.program-content i {
+  width: 25%;
+  display: flex;
+  justify-content: center;
+  transform: scale(2.5);
+  color: var(--clr-tertiary);
+  margin-left: 7px;
+}
+
+.program-content h2 {
+  color: var(--clr-secondary);
+  text-transform: capitalize;
+  font-family: var(--ff-primary);
+  font-size: 18px;
+  font-weight: 700;
+  text-align: left;
+  width: 75%;
+  width: 40%;
+}
+
+.program-btn {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.program-btn button {
+  background-color: var(--clr-secondary);
+  outline: none;
+  border: none;
+  width: 80%;
+  height: 80px;
+  padding: 10px 7px;
+  color: #fff;
+  font-size: 20px;
+  text-transform: capitalize;
+}
+
+.speakers {
+  width: 100%;
+  position: relative;
+  background-color: #fff;
+}
+
+.speakers-container {
+  width: 88%;
+  position: relative;
+  margin: 50px auto;
+  display: flex;
+  flex-direction: column;
+  row-gap: 45px;
+}
+
+.speaker-header {
+  display: flex;
+  position: relative;
+  width: 100%;
+  align-items: center;
+  flex-direction: column;
+  row-gap: 15px;
+}
+
+.speaker-header h2 {
+  color: var(--clr-primary);
+  font-family: var(--ff-primary);
+  font-size: 20px;
+  text-transform: capitalize;
+  font-weight: 700;
+}
+
+.featured-speakers {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  row-gap: 35px;
+  width: 100%;
+  flex-wrap: wrap;
+}
+
+.speaker1 {
+  width: 100%;
+  position: relative;
+  display: flex;
+  column-gap: 5%;
+  justify-content: center;
+  margin-top: 20px;
+  background-color: rgba(243, 226, 226, 0.5);
+}
+
+.speaker1 img {
+  width: 100%;
+  height: 65%;
+  display: block;
+  position: absolute;
+  bottom: 15%;
+}
+
+.speaker-img-cont {
+  width: 150px;
+  position: relative;
+  align-items: flex-end;
+  z-index: 1;
+}
+
+.speaker-img-cont::before {
+  content: "";
+  position: absolute;
+  width: 70%;
+  height: 30%;
+  top: 5%;
+  left: -10%;
+  z-index: -1;
+  background-color: var(--clr-primary);
+}
+
+.speaker-content {
+  width: 55%;
+  display: flex;
+  flex-direction: column;
+}
+
+.speaker-content h2 {
+  position: relative;
+  margin-top: 10px;
+  width: 100%;
+  font-family: var(--ff-primary);
+  color: var(--clr-primary);
+  text-transform: capitalize;
+  font-size: 18px;
+}
+
+.speaker-edu {
+  position: relative;
+  margin: 5px 0;
+  width: 100%;
+  color: var(--clr-secondary);
+  text-transform: capitalize;
+  font-style: italic;
+}
+
+.small-under {
+  outline: 1px solid var(--clr-tertiary);
+  margin-bottom: 40px;
+}
+
+.speaker-text {
+  position: relative;
+  width: 100%;
+  color: var(--clr-primary);
```

Comment:
- [ ] As pointed out by the previous reviewer, please use a lighter text as it is in the original template. This is what your page currently looks like.
![Screenshot 2021-10-13 at 10-53-47 Overcoming the Portal ](https://user-images.githubusercontent.com/70150423/137119916-a0ed9816-67e9-4e80-9561-a808e0fcbb98.png)